app/recordtransfer/admin.py

```diff
@@ -202,9 +202,29 @@ class UploadSessionAdmin(ReadOnlyAdmin):
     def upload_size(self, obj):
         return get_human_readable_size(obj.upload_size, 1000, 2)
 
-    fields = ["token", linkify("user"), "started_at", "file_count", "upload_size", "status"]
+    fields = [
+        "token",
+        linkify("user"),
+        "started_at",
+        "file_count",
+        "upload_size",
+        "status",
+        "last_upload_interaction_time",
+        "expired",
+        "expires_at",
+    ]
```

Comment:
Grouping the date times together makes this easier to read.

Since this is a readonly admin, I think we're OK to include the shorter name for the last upload time like I suggested in the `list_display`.

```suggestion
    fields = [
        "token",
        linkify("user"),
        "started_at",
        "last_upload_at",
        "file_count",
        "upload_size",
        "status",
        "expired",
        "expires_at",
    ]
```

Reply:
I agree with everything.

Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/403/commits/d6e1a7775637863e5d2fae36c914090a98fd7b03